packages/amplication-client/src/routes/appRoutes.tsx

```diff
@@ -178,5 +179,12 @@ export const Routes: RouteDef[] = [
     routeTrackType: "signup",
     exactPath: true,
     isAnalytics: true,
+  },
+      // Fallback route for wrong paths
+  {
+    path: "*",
+    Component: NotFoundPage,
```

Comment:
There are TS errors when using this code since the Component property expects to get a type of  LazyExoticComponent<ComponentType<any>>;

instead use this code (also remove the import at the begging of the page)

same for all the files below

```suggestion
    Component: lazy(() => import("../404/NotFoundPage")),
```